src/resources/Pipelines/Conditions/Condition.ts

```diff
@@ -36,4 +36,17 @@ export default class Condition extends Resource {
             conditionModel
         );
     }
+
+    bulkGet(conditionIds: string[], params: ListConditionsOptions = {}) {
```

Comment:
I don't know if you want to document that anywhere but we support a max of 1000 incoming ids.

Reply:
- [ ] JSDOC